wpilibc/src/main/native/cpp/motorcontrol/PWMMotorController.cpp

```diff
@@ -37,6 +37,10 @@ double PWMMotorController::Get() const {
   return m_pwm.GetSpeed() * (m_isInverted ? -1.0 : 1.0);
 }
 
+units::volt_t MotorController::GetVoltage() const {
```

Comment:
Add a prototype for this function to PWMMotorController.h (along with docs), and rename replace `MotorController::` with `PWMMotorController::`.

Reply:
Aready had a prototype with docs